config/application.yml.default

```diff
@@ -143,7 +143,7 @@ irs_attempt_api_audience: 'https://irs.gov'
 irs_attempt_api_auth_tokens: ''
 irs_attempt_api_csp_id: 'LOGIN.gov'
 irs_attempt_api_enabled: false
-irs_attempt_api_aws_s3_enabled: false
+irs_attempt_api_aws_s3_enabled: true
```

Comment:
this will enable the feature in *all* environments, not just `dev` like the description says

Reply:
I feel like not all environments will have a correctly configured bucket (multiple personal sandboxes, etc) so I would probably just manually enable in the "key" environments (dev, int, staging, prod)

Reply:
No worries. This is all a learning process for me. I just appreciate the help :). Just pushed a change to revert the flag to false except for the one under the 'development' section.